pontoon/terminology/views.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import absolute_import
+
+from django.http import JsonResponse
+from django.shortcuts import get_object_or_404
+from django.utils.datastructures import MultiValueDictKeyError
+
+from pontoon.base.models import Locale
+from pontoon.base.utils import require_AJAX
+from pontoon.terminology.models import Term
+
+
+@require_AJAX
+def get_terms(request):
+    """Retrieve terms for given source string and Locale."""
+    try:
+        source_string = request.GET["source_string"]
+        locale = request.GET["locale"]
```

Comment:
Nit: I'd rename this to `locale_code` to avoid having the `locale` variable contain two wildly different types of values. *(Gosh look at me I'm becoming a TypeScript developer! :scream: )*